doc.go

```diff
@@ -36,6 +36,7 @@ func ConvertDoc(r io.Reader) (string, map[string]string, error) {
 		doc, err := mscfb.New(f)
 		if err != nil {
 			log.Printf("ConvertDoc: could not read doc: %v", err)
+			return
```

Comment:
When we hit this case we will cause `ConvertDoc` to block forever, is that intentional? I suppose it's no worse than what happens now.

Reply:
Yeah I wondered about it. I think it's the same as what is already happening:

- trying to use `doc` causes panic
- recover in the defer catches it and converts it to a log statement
- implied `return` happens just as my explicit `return` in this commit happens

Reply:
I suppose my change is to just make it clearer the return was missed rather than fix underlying issues.

Reply:
Yep, SGTM.